package.json

```diff
@@ -51,6 +51,7 @@
     "@angular/platform-server": "4.1.0",
     "@types/jasmine": "2.6.0",
     "@types/node": "8.0.1",
+    "@kaiu/serializer": "^1.1.0",
```

Comment:
For devDependencies, I think it should be better to restrict more the version range. Like full strict `1.1.0` or at least `~1.1.0`. I have to many repo that broke (several months later) because of it's devDependencies being too permissive.